spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -1318,8 +1324,16 @@ public void createViewWithSubqueryExpressionInFilterThatIsRewritten()
     sql("USE spark_catalog");
 
     assertThatThrownBy(() -> sql(sql))
-        .isInstanceOf(AnalysisException.class)
-        .hasMessageContaining(String.format("The table or view `%s` cannot be found", tableName));
+        .satisfiesAnyOf(
+            throwable ->
+                assertThat(throwable)
```

Comment:
Hive throws Exception of type `SparkException` with error message as `Exception thrown in awaitResult`. 
Should we override these tests?